idunn/utils/result_filter.py

```diff
@@ -0,0 +1,183 @@
+import logging
+import re
+from typing import List
+from unidecode import unidecode
+
+from jellyfish import damerau_levenshtein_distance
+
+
+logger = logging.getLogger(__name__)
+
+
+# Typical suffixes found after numbers such as "4bis", "4th", ...
+NUM_SUFFIXES = ["bis", "ter", "quad", "e", "è", "eme", "ème", "er", "st", "nd", "rd", "th"]
+
+# Abrevation typical found in places, addresses, ...
+ABREVIATIONS = {
+    "av": "avenue",
+    "bd": "boulevard",
+    "r": "rue",
+    "st": "saint",
+}
```

Comment:
A few more items could be imported from https://github.com/CanalTP/mimirsbrunn/blob/master/libs/mimir/src/rubber.rs#L55-L60